kubernetes/token.go

```diff
@@ -1,25 +1,91 @@
 package kubernetes
 
-import "io/ioutil"
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/kiali/kiali/log"
+)
 
 // Be careful with how you use this token. This is the Kiali Service Account token, not the user token.
 // We need the Service Account token to access third-party in-cluster services (e.g. Grafana).
 
-const DefaultServiceAccountPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+var DefaultServiceAccountPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+var refreshTime = int64(60)
```

Comment:
Need we to make this configurable, since the refreshTime is different in different platforms?

Reply:
Thanks for your comment, @Xunzhuo 
I'm reviewing this.